src/BloomExe/Edit/EditingView.cs

```diff
@@ -295,29 +293,16 @@ private void SetupThumnailLists()
 		{
 			_pageListView.Dock = DockStyle.Fill;
 			_pageListView.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
-			_templatePagesView.BackColor = _pageListView.BackColor = _splitContainer1.Panel1.BackColor;
 			_splitContainer1.Panel1.Controls.Add(_pageListView);
-
-			_templatePagesView.Dock = DockStyle.Fill;
-			_templatePagesView.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowAndShrink;
 		}
 
-
+		// TODO: this _splitContainer2 could be eliminated now that we no longer have the TemplatePagesView
 		private void SetTranslationPanelVisibility()
 		{
 			_splitContainer2.Panel2.Controls.Clear();
 			_splitTemplateAndSource.Panel1.Controls.Clear();
 			_splitTemplateAndSource.Panel2.Controls.Clear();
-
-			if (_model.ShowTemplatePanel) //Templates only
-			{
-				_splitContainer2.Panel2Collapsed = false;
-				_splitContainer2.Panel2.Controls.Add(_templatePagesView);
-			}
-			else
-			{
-				_splitContainer2.Panel2Collapsed = true;
-			}
+			_splitContainer2.Panel2Collapsed = true; // used to hold TemplatesPagesView
```

Comment:
spec called for it to be visible but greyed out when translating. we do that elsewhere.